dotCMS/src/main/java/com/dotcms/contenttype/business/FieldAPI.java

```diff
@@ -229,4 +220,37 @@ default FieldAPI instance() {
 	 * @throws DotDataException
 	 */
     void saveFields(final List<Field> fields, final User user) throws DotSecurityException, DotDataException;
+
+	/**
+	 * Apply Internationalization to field property using a Language Variable, for each field property set into
+	 * fieldMap a Language Variable with the name contentTypeVariable.fieldVariable.propertyName is searched, if it
+	 * exists then the field's property value is replaced by the language variable value
+	 *
+	 * @param contentType
+	 * @param contentTypeInternationalization set the mode, language and user to search the Language Variable
+	 * @param fieldMap field properties
+	 * @return  The Field's properties with the new internationalization values
+	 */
+	public Map<String, Object> getFieldInternationalization(
+			final ContentType contentType,
+			final ContentTypeInternationalization contentTypeInternationalization,
+			final Map<String, Object> fieldMap
+	);
+
+	/**
+	 * Apply Internationalization to field property using a Language Variable, for each field property set into
+	 * fieldMap a Language Variable with the name contentTypeVariable.fieldVariable.propertyName is searched, if it
+	 * exists then the field's property value is replaced by the language variable value
+	 *
+	 * @param contentType
+	 * @param contentTypeInternationalization set the mode, language and user to search the Language Variable
+	 * @param fieldMap field properties
+	 * @return  The Field's properties with the new internationalization values
+	 */
+	public Map<String, Object> getFieldInternationalization(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary modifier 'public' on method 'getFieldInternationalization': the method is declared in an interface type](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5323537)